### PR TITLE
ci: migrate release from docker hub into github container registry

### PR DESCRIPTION
**Pull request by scbizu**
Signed-off-by: scnace <scbizu@gmail.com>

Per #391 

- [ ] Custom S3 bucket to get.helm.sh
- [ ] Docker Hub to GitHub registry

### CODE REVIEW

File: .github/workflows/mig.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+name: mig-test
+
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (jdolitsky) [inline .github/workflows/mig.yml:6]: Switch to build on push to main:
```
on:
  push:
    branches: main
    tags:
      - v*
```

GitHub actions will not use your the repo secrets on pull request, so we will just need to merge it to test

**Review by jdolitsky — CHANGES_REQUESTED** [review]
I'm not sure we need personal access token.. but we can remove later

File: .github/workflows/mig.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      - name: run unit tests
+        run: make test
+      - name: build binary
+        run: make build-linux
+      - name: run acceptance tests
+        run: sudo pip install virtualenv && make acceptance
+      - name: Prepare
+        id: prepare
+        run: |
+          DOCKER_IMAGE=chartmuseum/chartmuseum
```
Comment (jdolitsky) [inline .github/workflows/mig.yml:29]: I think the issue is this probably needs to be `ghcr.io/helm/chartmuseum`

File: .github/workflows/mig.yml (hunk 1 above)
Comment (scbizu) [reply .github/workflows/mig.yml]: Sadly :(

File: .github/workflows/mig.yml (hunk 2 above)
Comment (scbizu) [reply .github/workflows/mig.yml]: okay i see

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**